src/main/java/command/IgnoreCommand.java

```diff
@@ -15,14 +15,9 @@ public IgnoreCommand(int index) {
 
     @Override
     public void execute(TaskList tasks, Storage storage) throws DukeException {
-        if (indexOfTask < 0 || indexOfTask > (tasks.getSize() - 1)) {
-            throw new DukeException(DukeException.taskDoesNotExist());
-        }
```

Comment:
This if-statement is needed to ensure an out-of-bound integer not be passed into the TaskList to mark a task as ignored
Can I know why this part needs to be removed?

Reply:
Accidental removal,will put back